app/modules/incident/incident_helper.py

```diff
@@ -787,3 +796,37 @@ def return_channel_name(input_str):
     if input_str.startswith(prefix):
         return "#" + input_str[len(prefix) :]
     return input_str
+
+
+def fetch_user_details(client, channel_id):
```

Comment:
Since this is specific to Slack integration, do you think we could move it to the integrations/slack/channels so that we can reuse it across the whole app?

Reply:
Good point! I will move it.

Reply:
This has been done! Thanks for the suggestion.